README.md

```diff
@@ -23,6 +18,11 @@ Check it out right now at [http://fountainjs.io/](http://fountainjs.io/).
 
 > Gulp provide fast workspace with quick feedback.
 
+# **Unmaintained**, **Outdated** ?
+Yup, this generator works but is unmaintained and outdated [for various reasons](https://github.com/Swiip/generator-gulp-angular/pull/1155).
+But don't panic the most important features and more are present in [FountainJS](https://github.com/FountainJS/generator-fountain-webapp).
+
+You can find a use case in the [Yeoman codelab](http://yeoman.io/codelab/).
 
```

Comment:
In fact, I don't find more than you how to explain why it's outdated. The only thing which pains me is that the redirection to Fountain is not visible enough.

Can you simply keep the warning you have and the bring back the Fountain anouncement we have now and add the logo perhaps?